demo/tsconfig.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Can copy this nicer tsconfig: https://github.com/GoogleChrome/rendertron/blob/master/tsconfig.json